pkg/aws/service.go

```diff
@@ -45,28 +47,40 @@ func mustGetEnv(envVar string) string {
 	return value
 }
 
+func mustGetInt(envVar string) int64 {
+	stringValue := mustGetEnv(envVar)
+	value, err := strconv.ParseInt(stringValue, 10, 64)
+	if err != nil {
+		panic(fmt.Errorf("parsing env var %s to int: %w", envVar, err))
```

Comment:
suggestion: we may want to include the `stringValue` returned that was expected to be an int, although I suspect this is a _very_ rare panic.